docs/troubleshooting-guide.md

```diff
@@ -183,8 +183,21 @@ Furthermore, you might see tags such as `${test.student1}` and `${version}` in s
 
 1. **ERROR (on Linux)**: `java.io.IOException: Directory "/tmpfiles" could not be created`.
 
-   **SOLUTION**: Add `-Djava.io.tmpdir=/path/to/teammates/tmp` for the tests' run configurations. The "tmp" folder in the specified directory needs to be created before running the tests.
-
+  **SOLUTION**: Add `-Djava.io.tmpdir=/path/to/teammates/tmp` for the tests' run configurations. The "tmp" folder in the specified directory needs to be created before running the tests.
+
+  1. **ERROR**: Error message that running dev server :
```

Comment:
Maybe replace **ERROR** with **ERROR (on IntelliJ)** so that it is easier for other people to search IntelliJ specific issues!